MediaRecorder.html

```diff
@@ -724,14 +724,14 @@
           <dt><dfn><code>timecode</code></dfn> of type <span class "idlAttrType">
           <a href="https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">
           DOMHighResTimeStamp</a></span>, readonly</dt>
-          <dd>Timecode associated with <var>data</var>.
-          The timestamp MUST be relative to the first produced
-          <a>BlobEvents</a>'s <a>timecode</a> (note that this value does not
-          need to be zero).
+          <dd>The difference between the current <var>data</var>'s timestamp and
+          the <var>timecode</var> of the first produced <a>BlobEvent</a>.
```

Comment:
s/timecode/timestamp/ here, otherwise the definition seems recursive and wrong (isn't first timecode ~0)?

I think the terminology here has to be that chunks have timestamps (relative to some common origin of no importance) and that`timecode = timestampB - timestampA`. Do I have the intent right?